opt_einsum/contract.py

```diff
@@ -303,7 +309,12 @@ def contract_path(*operands, **kwargs):
 
         einsum_str = ",".join(tmp_inputs) + "->" + idx_result
 
-        contraction = (contract_inds, idx_removed, einsum_str, input_list[:], do_blas)
+        if len(input_list) <= 10:
```

Comment:
Any harm in extending this to 15-20?

Reply:
None memory wise! 10 was just a rough judgement call on neatness, but 20 feels fine as well.